src/ploneorg/addonlisting/browser/configure.zcml

```diff
@@ -43,6 +43,15 @@
       template="templates/addon_base_view.pt"
       permission="zope2.View"
       />
+
+  <browser:page
+      name="AddOnFolderView"
+      for="*"
+      layer="zope.interface.Interface"
```

Comment:
last open point on the review list to be updated, as soon as you have changed and changed from WIP to Ready, I will approve and merge